allennlp/nn/beam_search.py

```diff
@@ -14,11 +12,8 @@
 class BeamSearch:
     """
     Implements the beam search algorithm for decoding the most likely sequences.
-
```

Comment:
All of the blank lines are here for a reason. The API docs won't render properly without them.

Reply:
Fix